lib/asciidoctor/pdf/converter.rb

```diff
@@ -2180,10 +2180,23 @@ def convert_table node
           head_border_bottom_width = theme.table_head_border_bottom_width || table_border_width
         end
         [:top, :bottom, :left, :right].each {|edge| border_width[edge] = table_border_width }
+
         table_grid_color = theme.table_grid_color || table_border_color
-        table_grid_style = (theme.table_grid_style || table_border_style).to_sym
+        table_grid_style = theme.table_grid_style || table_border_style
+        if table_grid_style.is_a? Array
+          raise 'table-grid-style array must contain two styles' if table_grid_style.size != 2
+          table_grid_style = [table_grid_style[0].to_sym, table_grid_style[1].to_sym]
+        else
+          table_grid_style = Array.new 2, table_grid_style.to_sym
```

Comment:
This can be simplified to:

```
table_grid_style = [table_grid_style.to_sym]
```

Prawn already understands the single-entry array.